www/src/components/sidebar/accordion.js

```diff
@@ -139,42 +139,44 @@ class Accordion extends React.Component {
           uid={uid}
           disableAccordions={disableAccordions}
         />
-        <ul
-          id={uid}
-          sx={{
-            display: isExpanded ? `block` : `none`,
-            listStyle: `none`,
-            margin: 0,
-            position: `relative`,
-            ...(item.ui === `steps` && {
-              "&:after": {
-                backgroundColor: `ui.border`,
-                bottom: 0,
-                content: `''`,
-                left: 27,
-                position: `absolute`,
-                top: 0,
-                width: 1,
-              },
-            }),
-          }}
-        >
-          {item.items.map(subitem => (
-            <Item
-              activeItemLink={activeItemLink}
-              activeItemParents={activeItemParents}
-              createLink={createLink}
-              item={subitem}
-              key={subitem.title}
-              location={location}
-              onLinkClick={onLinkClick}
-              isExpanded={isExpanded}
-              onSectionTitleClick={onSectionTitleClick}
-              openSectionHash={openSectionHash}
-              ui={item.ui}
-            />
-          ))}
-        </ul>
+        {isExpanded && (
+          <ul
+            id={uid}
+            sx={{
+              // display: isExpanded ? `block` : `none`,
```

Comment:
I changed this to get the test working. @fk do you have any context for why this was implemented using `display` instead of a conditional? (e.g. performance)

`react-testing-library` does have a `.toBeVisible()` matcher that checks for `display: none`, but it doesn't work in JSDOM: https://twitter.com/kentcdodds/status/1051830222322991105

Reply:
> do you have any context for why this was implemented using display instead of a conditional?

Most likely because I'm old, and I'm subconciously still optimizing for IE8 ;) LOL
Seriously—no clue. Not fully embracing/having understood React and thinking in (very) old patterns is my strongest guess … or, since I took over most of this from the React docs, it might have been implemented like this over there and I copy-pasta'ed? 🤷‍♂️

Reply:
Looking at the build, it does feel like loading pages is much faster — most likely because we don't have to render the *entire* sidebar DOM tree each time if the sidebar isn't fully expanded:

* This branch: https://build-be8b047e-6a17-400c-b09a-a27e56d5d0da.gtsb.io/docs/
* master: https://gatsbyjs.org/docs/